frontend/src/app/accountCreation/MfaComplete/MfaComplete.tsx

```diff
@@ -25,6 +25,7 @@ export const MfaComplete = () => {
           className="margin-top-3"
           label={"Get started"}
           type={"submit"}
+          onClick={() => (window.location.pathname = "/")}
```

Comment:
For now, I just set the "Get started" button on the final page to redirect to the home page of the static site. Once we implement the in-app login flow, I will change this to redirect to `/login` or whatever we end up using